### ISSUE TITLE
vbuild 4.3.3 not loading mdi (material design icons) font

### ISSUE THREAD

**Issue by layanto** [egoist/vbuild#87]
With vbuild 4.3.2, all working. No code change and only upgrade vbuild to 4.3.3, mdi font is not loaded in run dev.

**Comment by layanto** [comment]
Also not working in vbuild 4.3.4

**Comment by layanto** [comment]
Reverting this commit https://github.com/egoist/vbuild/commit/d58bbd8d6d38d9ee0801ef6495d2ffc0f97796e1
fixes this issue

**Comment by egoist** [comment]
use `file-loader` to load svg if you like.

**Comment by layanto** [comment]
Actually I am not using svg. With vbuild v4.3.2, Chrome actually loads woff2 and not the svg. I load the mdi's css via 
```
import 'mdi/css/materialdesignicons.css'
```
mdi's css includes the font file or svg as follows
```
src: url("../fonts/materialdesignicons-webfont.eot?v=1.7.22");
src: url("../fonts/materialdesignicons-webfont.eot?#iefix&v=1.7.22") format("embedded-opentype"), url("../fonts/materialdesignicons-webfont.woff2?v=1.7.22") format("woff2"), url("../fonts/materialdesignicons-webfont.woff?v=1.7.22") format("woff"), url("../fonts/materialdesignicons-webfont.ttf?v=1.7.22") format("truetype"), url("../fonts/materialdesignicons-webfont.svg?v=1.7.22#materialdesigniconsregular") format("svg");
```

**Comment by egoist** [comment]
ah, maybe we have to revert that commit now

**Comment by egoist** [comment]
fixed in `v4.3.5`
